cluster/store/store.go

```diff
@@ -195,42 +195,12 @@ func (st *Store) Open(ctx context.Context) (err error) {
 	}
 	defer func() { st.open.Store(err == nil) }()
 
-	if err = os.MkdirAll(st.raftDir, 0o755); err != nil {
-		return fmt.Errorf("mkdir %s: %w", st.raftDir, err)
-	}
-
-	// log store
-	st.logStore, err = raftbolt.NewBoltStore(filepath.Join(st.raftDir, raftDBName))
-	if err != nil {
-		return fmt.Errorf("raft: bolt db: %w", err)
-	}
-
 	// log cache
-	logCache, err := raft.NewLogCache(logCacheCapacity, st.logStore)
+	logCache, err := st.init()
```

Comment:
**suggestion (if-minor):**  what do you think if we call it `initStores()` and make logCache part in the store which doesn't need to be returned, instead it will return error only ?

Reply:
i think the func now does more than initiating `logCache` , you don't think that could be confusing ?